piptools/resolver.py

```diff
@@ -629,7 +640,9 @@ def _do_resolve(
         compatible_existing_constraints: dict[str, InstallRequirement],
     ) -> bool:
         """
-        Return true on successful resolution, otherwise remove problematic
+        Actual resolution process.
+
+        Return True on successful resolution, otherwise remove problematic
```

Comment:
Things like this are linkable
```suggestion
        Return :py:data:`True` on successful resolution, otherwise remove problematic
```